cmd/mayactl/app/command/stats_helper_test.go

```diff
@@ -81,12 +82,22 @@ func TestGetVolDetails(t *testing.T) {
 			err:  util.PageNotFound,
 			addr: "MAPI_ADDR",
 		},
+		"OpenebsNamespacedVolume": {
+			volumeName: "vol",
+			namespace:  "",
```

Comment:
It will be good to verify one another test case for the any namespace other than  `""`.

Reply:
Its there.